lib/resty/core/phase.lua

```diff
@@ -0,0 +1,55 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
+local FFI_ERROR = base.FFI_ERROR
+local tostring = tostring
+
+ffi.cdef[[
+int ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
```

Comment:
Similar to other comments wrt. multiple style in multiple modules in the project, I can't seem to find a consistent style, even in regard to `ngx` vs `resty` modules; in both cases, some modules exhibit one style, some modules exhibit another.